source/HermitStore/DbContext.cs

```diff
@@ -37,7 +37,7 @@ public class CommunityDto
     {
         public required string community_name { get; set; }
         public string? community_image { get; set; }
-
+        public int? community_members { get; set; }
```

Comment:
Eftersom boten alltid vet hur många members communityn kommer ha borde vi sätta det här som en required attribute. Jag lägger in det.